.gitignore

```diff
@@ -193,4 +193,6 @@ privates_config.lua
 /out/
 
 # python files
-*.pyc
\ No newline at end of file
+*.pyc
+
+.idea/

```

Comment:
What is this and why would it be needed? Why don't you just configure whatever an 'idea' is to not feel the need to write its 'idea' directory?

Reply:
`idea` is generated intellij IDE